controller.js

```diff
@@ -23,5 +23,13 @@ const getReviewById = (request, response, next) =>{
     }).catch(next)
 }
 
+const getComments = (request, response, next) =>{
+    const reviewId = request.params.review_id
+    fetchComments(reviewId).then((comments)=>{
+        response.status(200).send(comments)
+    })
+    .catch(next)
+}
+
```

Comment:
I'd consider using object destructuring here eg:
const {review_id} = request.params
this is doing the same operation as line 27 but using the existing name of the key as the variable.